tiles/src/main/java/com/protomaps/basemap/layers/Landuse.java

```diff
@@ -21,14 +21,14 @@ public void processFeature(SourceFeature sf, FeatureCollector features) {
       sf.hasTag("landuse", "recreation_ground", "industrial", "brownfield", "railway", "cemetery", "commercial",
         "grass", "orchard", "farmland", "farmyard", "residential", "military") ||
       sf.hasTag("leisure", "park", "garden", "golf_course", "dog_park", "playground", "pitch", "nature_reserve") ||
-      sf.hasTag("man_made", "pier") ||
+      sf.hasTag("man_made", "pier") || sf.hasTag("man_made", "bridge") ||
```

Comment:
Nit, since these are both the same tag key, rewrite asa list of values, like:

>       sf.hasTag("man_made", "pier", "bridge") ||

Reply:
Agree that in the case of `bridge` it's assumed to be a polygon per the [wiki](https://wiki.openstreetmap.org/wiki/Tag:man_made%3Dbridge) and in practice.

Reply:
Does this need a style change, too? Something like `id: "landuse_pedestrian",` either duplicating that style layer or adding `bridge` to it's the filter?

Reply:
It's covered here: https://github.com/protomaps/basemaps/blob/main/styles/src/base_layers.ts#L286

Reply:
Can you include a before/after screenshot showing the data change in the style? 

(I think that style layer filter needs a logic change...)

Reply:
Will provide visual test link when the 3.0.1 build completes tomorrow... think the style with `pmap:kind=pedestrian` is good enough